src/core/scripting.c

```diff
@@ -1343,6 +1343,8 @@ static uint8_t _readLuminance(struct GBALuminanceSource* luminance) {
 }
 #endif
 
+#pragma push_macro("CALLBACK")
```

Comment:
I have no idea what warning this fixes. Is it Windows-specific? Regardless, it doesn't belong in this PR.